frontend/src/upload.js

```diff
@@ -346,18 +501,67 @@ $(document).ready(function() {
     row.appendChild(del);
     $('tbody').append(row); //add row to table
 
+    let unexpiredFiles = 0;
+
+    for (let i = 0; i < localStorage.length; i++) {
+      const id = localStorage.key(i);
+      if (id !== 'totalUploads' && 
+          id !== 'totalDownloads' &&
+          id !== 'referrer') {
+          unexpiredFiles += 1;
+      }
+    }
+
+    let totalDownloads = 0;
+    if (localStorage.hasOwnProperty('totalDownloads')) {
+      totalDownloads = localStorage.getItem('totalDownloads');
+    }
+
     // delete file
     $popupText.find('.del-file').click(e => {
       FileSender.delete(file.fileId, file.deleteToken).then(() => {
         $(e.target).parents('tr').remove();
-        localStorage.removeItem(file.fileId);
+        const timeToExpiry = 86400000 - (new Date().getTime() - file.creationDate.getTime());
```

Comment:
Not sure if we want to convert the magic 86400000 number to a `const`. For example:
```js
const timeToExpiry = ONE_DAY_IN_MS - Date.now() - file.creationDate.getTime();
```